test/e2e/storage/csi_mock_volume.go

```diff
@@ -1301,8 +1301,7 @@ var _ = utils.SIGDescribe("CSI mock volume", func() {
 						framework.ExpectNoError(err, "Failed to get claim: %v", err)
 					}
 					framework.Logf("PVC not found. Continuing to test VolumeSnapshotContent finalizer")
```

Comment:
Alternatively, is it reasonable to `framework.Failf()` if we get back *any* error (including not found)?

Reply:
I think the test here is that if we get not found error, the test will continue. So not found err is part of the expecting scenario that we can encounter